### DISCUSSION TITLE
How to remove all local copies of files in local and still use the app to sync timely additions to folders?

### DISCUSSION THREAD

**Discussion by Eduardo-HortaS** [abraunegg/onedrive#2443] [Q&A] [answered]
Hey, thanks for your work, I'm struggling a bit to understand how to do this, I plan on using onedriver for when I want to use something thats on remote locally and this app just to sync some additions that I eventually make, is this a reasonable use or is there a better way to go about this? Thanks!

**Comment by abraunegg** [comment, marked as answer]
> is this a reasonable use or is there a better way to go about this?

No this is not reasonable.

Pick either 'onedriver' or 'onedrive' .. do not use or try and use both at the same time.

> How to remove all local copies of files in local and still use the app to sync timely additions to folders?

Removing local data, without using the correct configuration, will remove your data online. Please read the usage document on configuration options.
